rpmlint/descriptions/CheckBuildRoot.toml

```diff
@@ -0,0 +1,2 @@
+file-contains-buildroot="""
+Your file contains traces of $RPM_BUILD_ROOT."""
```

Comment:
maybe it shoudl mention the %{buildroot} instead of $RPM_BUILD_ROOT as the later is the deprecated bash var name.